certbot/cli.py

```diff
@@ -532,6 +532,7 @@ def parse_args(self):
         parsed_args = self.parser.parse_args(self.args)
         parsed_args.func = self.VERBS[self.verb]
         parsed_args.verb = self.verb
+        parsed_args.renewing = "unknown"  # an important config property we don't know until later
```

Comment:
I've explained what it's used for *and* removed the claim that it's important :)